SharpAdbClient/AdbSocket.cs

```diff
@@ -402,7 +402,7 @@ public Stream GetShellStream()
         /// <remarks>
         /// This uses the default time out value.
         /// </remarks>
-        protected bool Write(byte[] data)
+        internal bool Write(byte[] data)
```

Comment:
I don't think you're actually using this method, right? So can we keep it `protected`?

Reply:
Hi, you're right, I called this function, but I switched to

```
 (shellStream.Inner as System.Net.Sockets.NetworkStream).Write(bytes, 0, bytes.Length);
```
I'll correct it, thank you.